catalog.html

```diff
@@ -1,149 +1,164 @@
-                    </div>
-                </li>
-                <li>
-                    <img src="https://via.placeholder.com/360x620" alt="Демо-макет Mishka">
-                    <div>
-                        <h3>Mishka</h3>
-                        <p>Интернет-магазин вязаных игрушек</p>
-                        <a href="#">9 900 ₽</a>
-                    </div>
-                </li>
-                <li>
-                    <img src="https://via.placeholder.com/360x620" alt="Демо-макет A plus">
-                    <div>
-                        <h3>A plus</h3>
-                        <p>Лендинг курсов повышения квалификации</p>
-                        <a href="#">9 900 ₽</a>
-                    </div>
-                </li>
-                <li>
-                    <img src="https://via.placeholder.com/360x620" alt="Демо-макет Kvas">
-                    <div>
-                        <h3>Кваст</h3>
-                        <p>Промо-сайт производителякрафтового кваса</p>
-                        <a href="#">9 900 ₽</a>
-                    </div>
-                </li>
-            </ul>
-            </section>
-            <ul>
-                <li><a href=#>1</a></li>
-                <li><a href=#>2</a></li>
-                <li><a href=#>3</a></li>
-                <li><a href=#>Следующая</a></li>
-            </ul>
-      </main>
-      <footer>
-        <section>
-          <h2>Схема проезда</h2>
-          <p>
-            NЁRDS DESIGN STUDIO
-          </p>
-          <p>
-            191186, Санкт-Петербург,<br>
-            ул. Б. Конюшенная, д. 19/8<br>
-          </p>
-          <p>
-            тел. <a href="tel:+78122757575">+7 (812) 275-75-75</a>
-          </p>
-          <p>
-            <a href=#>Напишите нам</a>
-          </p>
-        </section>
-        <section>
-          <h2>Социальные сети</h2>
-          <ul>
-            <li><a href="">VK</a></li>
-            <li><a href="">Facebook</a></li>
-            <li><a href="">Instagram</a></li>
-          </ul>
-        </section>
-        <div>
-          <p>Давайте дружить, это выгодно!</p>
-          <p>Скидка 10% для друзей из социальных сетей.</p>
+    </section>
+    <!-- Тут будет фильтр стоимость/сетка/особенности -->
+
+    <section class="demos">
+      <h2 class="visually-hidden">Визуальный набор шаблонов</h2>
+      <ul class="demos-list">
+        <!-- Хочу сделать li контейнером, менять сзади фон при наведении, не думаю, что надо еще раз оборачивать в div -->
+        <!-- Обернул всплывающее окно в div, чтобы им легче было манипулировать (не до конца понимаю верно это или нет) -->
+        <li class="demos-item">
+          <div class="demo-img">
+          <img src="https://via.placeholder.com/360x620" alt="Демо-макет Sedona">
+        </div>
+          <!-- Обернул в div, чтобы отделить от основного куска (появляется при наведении) -->
+          <div class="demos-item-popup">
+            <h3>Sedona</h3>
```

Comment:
Нужно текст обернуть ссылкой. Вот так:
```suggestion
            <h3><a href="#">Sedona</a></h3>
```

То, что тут должна быть ссылка есть указание в [ТЗ](https://up.htmlacademy.ru/htmlcss/29/project/nerds#nerds-3-10). (Ну и собственно можно понять по стилям стайлгайда).